flow/Platform.actor.cpp

```diff
@@ -1933,6 +1934,18 @@ std::string epochsToGMTString(double epochs) {
 	return timeString;
 }
 
+std::vector<std::string> getEnvironmentKnobOptions() {
+	char** e = environ;
```

Comment:
Is this available on all platforms? I tried a quick search, but I couldn't tell for sure.

Reply:
I researched this before posting the PR. `getenv()` is cross platform (Mac/Windows/Linux) (see [1] and elsewhere), and is a wrapper around global `environ` I'm using here. I found a caveat in Windows. The mechanism is reliable at process start time but not after. If an environment variable changes after the process starts, you need to use a different MSFT API. [2] But I do not think that would be an issue for this use case.

[1] [stackoverview discussion on environ](https://stackoverflow.com/questions/21557479/how-to-get-environment-variables-in-c-in-a-cross-platform-way)
[2] [stackoverflow discussion on Windows](https://stackoverflow.com/questions/631664/accessing-environment-variables-in-c)